src/sets.jl

```diff
@@ -711,9 +711,9 @@ end
 """
     IndicatorSet{A, S <: AbstractScalarSet}(set::S)
 
-``\\{((y, x) \\in \\{0, 1\\} \\times \\mathbb{R}^n : y = 0 \\implies x \\in set\\}``
```

Comment:
you added opening parentheses, which is the closing one?

Reply:
Actually, I removed parentheses that were not closed anywhere.